pytest_jira.py

```diff
@@ -102,7 +102,7 @@ def pytest_runtest_setup(self, item):
 
         # Check all linked issues
         for issue_id in jira_ids:
-            if not jira_run and not self.is_issue_resolved(issue_id):
+            if not self.is_issue_resolved(issue_id) and not jira_run:
```

Comment:
Not exactly. Since the statement evaluates from left to right, the interpreter simply stops evaluating once the left side of the `and` is `True`. I thought it'd be suffice to fix test but I was wrong (it also broke a lot of stuff).

Adding tests and fixing this properly soon.